aspnet/fundamentals/app-state/sample/src/AppState/Model/RequestEntry.cs

```diff
@@ -1,11 +1,7 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace AppState.Model
 {
-    [Serializable]
```

Comment:
I could not find Serializable in the .NET Core. I was planning on asking someone about that in the PR but I forgot about it. Is there a .NET Core alternative that I am unaware of? Also for the future is there anywhere to go for finding information on the api? So far I have just been exploring intellisense in visual studio.